src/components/Forum/CreateItem.js

```diff
@@ -1,4 +1,4 @@
-import React, { useState } from 'react';
+import React from 'react';
 import {useFormValidation} from 'hooks';
 import {validateCreate} from 'services';
```

Comment:
spaces between curly bracktes and hooks should follow one convention, in some files they exist, in other not